### ISSUE TITLE
Getting geolocation data

### ISSUE THREAD

**Issue by alexbudin** [brilliantlabsAR/noa-assistant#22]
Hi guys,

Love the glasses idea. Dont have them yet, but i love the open source idea and i want to understand a few things on what is possible with them.

Is geolocation data accessible with the requests? Is that taken from the mobile phone app?

Thank you,

Alex

**Comment by trzy** [comment]
Yes. Check out the source code — it’s passed to the Noa assistant server from the phone. The server uses it for LLM and search context.Sent from my iPhoneOn May 23, 2024, at 2:44 AM, Alex Budin ***@***.***> wrote:﻿
Hi guys,
Love the glasses idea. Dont have them yet, but i love the open source idea and i want to understand a few things on what is possible with them.
Is geolocation data accessible with the requests? Is that taken from the mobile phone app?
Thank you,
Alex
